kd/audit-archive-cronjob.yaml

```diff
@@ -28,7 +28,7 @@ spec:
               runAsUser: 1000
             command: ["/bin/sh", "-ce"]
             args:
-              - 'http_status=$( curl -gkv -X POST -H "Authorization: Basic $(echo -n ${IP_API_SERVICE_AUTH} | base64)" ${PTTG_IP_API_URL}/archive ); if [[ $http_status -eq 200 ]]; then exit 0; else exit 1; fi'
+              - 'http_status=$( curl -gk -w "%{http_code}" -X POST -H "Authorization: Basic $(echo -n ${IP_API_SERVICE_AUTH} | base64)" ${PTTG_IP_API_URL}/archive ); if [ $http_status == 200 ]; then exit 0; else exit 1; fi'
```

Comment:
I think the if bit should either be [[ $http_status == 200 ]] or [ $http_status -eq 200 ]
http://tldp.org/LDP/abs/html/comparison-ops.html

Reply:
Actually I think the == is still wrong, even with double brackets.

Reply:
It seems that the if in sh is quite lenient:
`
sh-4.4$ blah=1
sh-4.4$ if [ $blah -eq 1 ]; then echo 'this works'; fi
this works
sh-4.4$ if [ $blah = 1 ]; then echo 'this works'; fi
this works
sh-4.4$ if [ $blah == 1 ]; then echo 'this works'; fi
this works
sh-4.4$ if [ $blah -eq "1" ]; then echo 'this works'; fi
this works
sh-4.4$ if [ "$blah" -eq "1" ]; then echo 'this works'; fi
this works
sh-4.4$ if [ $blah = 1 ]; then echo 'this works'; fi
this works
`
However I will go for integer comparison as that seems the "correct way".

Reply:
It seems that the if in sh is quite lenient:
```
sh-4.4$ blah=1
sh-4.4$ if [ $blah -eq 1 ]; then echo 'this works'; fi
this works
sh-4.4$ if [ $blah = 1 ]; then echo 'this works'; fi
this works
sh-4.4$ if [ $blah == 1 ]; then echo 'this works'; fi
this works
sh-4.4$ if [ $blah -eq "1" ]; then echo 'this works'; fi
this works
sh-4.4$ if [ "$blah" -eq "1" ]; then echo 'this works'; fi
this works
```
However I will go for integer comparison as that seems the "correct way".